packages/core/src/middleware/koa-csp-headers.ts

```diff
@@ -0,0 +1,38 @@
+import type { MiddlewareType } from 'koa';
+import helmet from 'koa-helmet';
+import type { IRouterParamContext } from 'koa-router';
+
+import { EnvSet } from '#src/env-set/index.js';
+
+export default function koaCspHeaders<StateT, ContextT extends IRouterParamContext, ResponseBodyT>(
```

Comment:
i'll take a look today

Reply:
Hold this pr, gonna further refactor this PR,  just realized there are other headers that need to be exclusively set or unset for Mainflow due to our terms in the iframe feature.